backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_registration.py

```diff
@@ -7,13 +7,15 @@
 
 from .. import TstFunction
 
-TEST_COMPACT = 'aslp'
+TEST_COMPACT_ABBR = 'aslp'
+TEST_COMPACT_NAME = 'Audiology and Speech Language Pathology'
 TEST_LICENSE_TYPE = 'speech-language pathologist'
 
 MOCK_SSN_LAST_FOUR = '1234'
 MOCK_GIVEN_NAME = 'Joe'
 MOCK_FAMILY_NAME = 'Dokes'
-MOCK_STATE = 'ky'
+MOCK_JURISDICTION_POSTAL_CODE = 'ky'
```

Comment:
Strictly speaking, a zip code is a kind of postal code, this is a postal abbreviation.